projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -887,7 +887,7 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      * ```
     */
     get virtualizationState(): IForOfState {
-        return this.dropdown.verticalScrollContainer.state;
+        return this.dropdown.virtDir.state;
```

Comment:
I feel like it could be left as public for easier access of the virtual directive in certain use cases (multiple igxforof directives in a single view when using the shorthand syntax, passing data through a pipe and wanting to access the data that is rendered (dropdown.virtDir.igxForOf))